app/src/main/java/com/course_project_autotrace/BasicCarInfo/BasicCarInfoView.java

```diff
@@ -0,0 +1,6 @@
+package com.course_project_autotrace.BasicCarInfo;
```

Comment:
Although I was being skeptical of the View File as it is quite short,  we still have to follow clean architecture